README.md

```diff
@@ -36,11 +36,21 @@ You can create a native executable using:
 mvn package -Pnative
 ```
 
-Or, if you don't have GraalVM installed, you can run the native executable build in a container using: 
+Or, if you don't have GraalVM installed, you can run the native executable build in a container using:
```

Comment:
This file is obsolete since we dropped the catalogs to be a HTTP service in favor of ConfigMaps.